govcd/edgegateway_lb_service_monitor.go

```diff
@@ -0,0 +1,187 @@
+package govcd
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// CreateLBServiceMonitor synchronously creates a load balancer service monitor based on mandatory fields
```

Comment:
Here `synchronously` missing context - what part is sync.

Reply:
This whole operation is synchronous. I could change it but I think it implies that whole operation is sync. Open to suggestions/more readable ideas though

Reply:
CreateLBServiceMonitor creates a load balancer service monitor based on mandatory fields. NSX API supports only synchronous calls.

Reply:
It could not necessarily be synchronous. (I think there is a possibility to do it async, but still only one operation could happen so there is no point at all to use it)
Anyway I got the point what you're after.

Reply:
Maybe it's better now.

Reply:
Thank you